File: test-goldens/java/class_with_methods.java
/**
 * Service for managing users.
 */
public class UserService {
    private UserRepository repo;
    private final Logger logger;

    public UserService(UserRepository repo, Logger logger) {
        this.repo = repo;
        this.logger = logger;
    }

    public User findUser(String id) {
        return this.repo.findById(id);
    }
}

File: test-goldens/csharp/class_with_methods.cs
/// Service for managing users.
public class UserService {
    private UserRepository repo;
    private readonly ILogger logger;

    public UserService(UserRepository repo, ILogger logger) {
        this.repo = repo;
        this.logger = logger;
    }

    public User FindUser(string id) {
        return this.repo.FindById(id);
    }
}
